modules/alternatives-to-storing-admin-secrets-in-kube-system.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp/manually-creating-iam-gcp.adoc
+
+[id="alternatives-to-storing-admin-secrets-in-kube-system.adoc_{context}"]
+= Alternatives to storing administrator-level secrets in the `kube-system` project
+
+The Cloud Credential Operator (CCO) manages cloud provider credentials as Kubernetes custom resource definitions (CRDs). By setting different values for the `credentialsMode` parameter in the `install-config.yaml` file, the CCO can be configured to suit the security requirements of your organization.
+
+If you prefer not to store an administrator-level credential secret in the cluster `kube-system` project, you can choose one of the following options when installing {product-title} on AWS:
+
+* *Manage cloud credentials manually*. You can set the `credentialsMode` for the CCO to `Manual` to manage cloud credentials manually. Using manual mode allows each cluster component to have only the permissions it requires, without storing an administrator-level credential in the cluster. You can also use this mode if your environment does not have connectivity to the AWS public IAM endpoint. However, you must manually reconcile permissions with new release images for every upgrade.
+
+* *Remove the administrator-level credential secret after installing {product-title} with mint mode*. You can remove or rotate the administrator-level credential after installing {product-title} 4.4 or later with the `mint` CCO credentials mode applied. This option requires the presence of the administrator-level credential during an installation, but the credential is not stored in the cluster permanently and does not need to be long-lived.
```

Comment:
Why state a minimum version here? Isn't this the docs for a version that already meets this minimum anyway?

Reply:
Good point. I have now removed the version reference.